### ISSUE TITLE
Player accessibility improvements

### ISSUE THREAD

**Issue by FrenjaminBanklin** [ucfopen/this-or-that-materia-widget#55]
With the understanding that we're trying to make accessible a widget that's more or less choosing the correct option between two images...
- The 'Start' button is still in the tab order after a play has begun.
  - Either remove the 'Start' button from the tab order after starting a play or, better yet, remove it from the DOM entirely.
- Question text is not reachable in the tab order, which makes it unavailable to screen readers.
  - Make sure the question text has a tab order.

**Comment by clpetersonucf** [comment]
One concern here is adding the question to the tab index: most documentation is pretty explicit about using tab for focusable elements and inputs, not non-interactive elements like question text. The more ideal solution likely involves using semantic DOM elements and labels to describe the text content as being the question text. For instructions or contextual information related to specific elements, focusable or otherwise, we can lean on things like `aria-describedby` or aria-labelledby`
